base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
I think I'd be ok with calling this `pipe()` and adding `pipepair()` in the future if necessary. Names that differ only in case are really annoying.